kotlinpoet-km/src/main/kotlin/com/squareup/kotlinpoet/km/Flags.kt

```diff
@@ -257,7 +257,9 @@ val ImmutableKmValueParameter.isNoInline: Boolean get() = Flag.ValueParameter.IS
 
 // Property flags.
 @KotlinPoetKm
-val Flags.isOverrideProperty: Boolean get() = Flag.Property.IS_FAKE_OVERRIDE(this)
+val Flags.isFakeOverrideProperty: Boolean get() = Flag.Property.IS_FAKE_OVERRIDE(this)
```

Comment:
From the doc: 
```
/**
 * A member kind flag, signifying that the corresponding function exists in the containing class because a function with a suitable
 * signature exists in a supertype. This flag is not written by the Kotlin compiler and its effects are unspecified.
 */
```